.github/workflows/twitter-together.yml

```diff
@@ -0,0 +1,41 @@
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - 'tweets/**'
+      - '**twitter-together.yml'
+
+  pull_request:
+    paths:
+      - 'tweets/**'
+      - '**twitter-together.yml'
+
+name: Twitter, together!
+jobs:
+  preview:
+    name: Preview
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
```

Comment:
having `if` clauses on both actions suggests breaking them into separate workflows: preview on PR, tweet on push

Reply:
we def could. This is mostly copy-pasted with minor edits from the twitter together docs

Reply:
Done. Should work -- initial setup just looks for a `tweets/` folder at the repo root.

Reply:
Annd reverted; we want the tweet to be dependent on the check job so they are validated before they are tweeted.

Reply:
And we should remove `if: github.event_name == 'pull_request'` if we want to make sure the preview always runs on both PRs and pushes